dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -149,6 +70,8 @@ const LastUpdated = ({
 	);
 };
 
+const formatTime = (time: number) => (time < 10 ? `0${time}` : time);
```

Comment:
There's a function in AR's `date` module called `padZero` that does almost exactly this 😄. Perhaps we should consider lifting and sharing? If you think that's outside the scope of this PR I can create a ticket.

Reply:
There's one being used in Source too `const pad = (n: number): number | string => n.toString().padStart(2, '0');` 😸 

There's probably a lot of this code in the internets.

Reply:
Sounds great, yes please! I'll add a TODO here 👍🏻

Reply:
Isn't time always going to be < 24 || < 60 ? In these instances, at least?

Reply:
@mxdvl you're right! That `number` type is a hangover from when we added [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)

Reply:
Also (and sorry to add to this thread but the best PRs are the ones with lots of comments on them!) maybe we can name this function `padString` (or similar)? `formatTime` suggests more is happening.

Reply:
Yes, sure - that seems reasonable